db/migrate/20200907122828_create_issues.rb

```diff
@@ -0,0 +1,14 @@
+class CreateIssues < ActiveRecord::Migration[6.0]
+  def change
+    create_table :issues do |t|
+      t.string :title
+      t.text :description
+
+      t.integer :issue_type # enum: bug or issue
+      t.integer :priority # enum: low or high
```

Comment:
Add company's association to all the tables + add association of project with issues once you have latest code

Reply:
ok

Reply:
Done